src/main/java/cinemas/controllers/user/MovieController.java

```diff
@@ -0,0 +1,24 @@
+package cinemas.controllers.user;
+
+import cinemas.models.Movie;
+import cinemas.services.MoviesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.List;
+
+@Controller
+public class MovieController {
+    @Autowired
+    private MoviesService moviesService;
+
+    @GetMapping("/movies/detail/{id}")
```

Comment:
```suggestion
    @GetMapping("/movies/{id}")
```
ri dc, thêm chữ detail là j, cơ mà method lại tên home thì hơi lạ, nó tên show sẽ hợp lý hơn